Assignment_Docs_Links.txt

```diff
@@ -0,0 +1,6 @@
+DBMS Session 1 Question 1 :
+https://drive.google.com/file/d/1Pm0CTz165gfbxyKkfydA2Gk-r268jMu-/view?usp=sharing (Flow Chart)
+https://drive.google.com/file/d/1E1K3_yaLxstEnqyZ9DEvNbsAo8QFcccR/view?usp=sharing (ER Diagram)
```

Comment:
Few points:

1. Identify the participating entities - list them
2. Identify the relations - list the relations
3. Identify the key attributes - list the attributes
4. then go for the ER diagram.

All the above listing can be done in a google doc.